README.md

```diff
@@ -34,12 +34,17 @@ Prerequisite: podman-compose or docker-compose.
 
 To start all dependencies and trustification components:
 
+1. Find the [latest version tag](https://github.com/trustification/trustification/releases)
+2. Set the environment variable to that version
+3. Start the containers
+
 ``` shell
 cd deploy/compose
-docker-compose -f compose.yaml -f compose-trustification.yaml -f compose-guac.yaml up
+export TRUST_VERSION=<version>
+docker-compose -f compose.yaml -f compose-trustification.yaml -f compose-guac.yaml -f compose-walkers.yaml up
 ```
```

Comment:
Can we add the spog url here, too? I wasn't sure what port to visit.

Reply:
It's in the .env file, I'd like to avoid having to set a bunch of env vars unless necessary (the version is necessary because we don't have 'latest').

Reply:
No, I meant just add a line of text telling them how to see what they just deployed, e.g. "Point your browser at http://localhost:8084 or whatever you set the SPOG_UI_PORT to in .env"

Also, why not 8080 instead of 8084? That might be the first thing they try anyway.

Reply:
Ah, I agree with that. The reason for 8080 I'm not sure, some of us have other processes occupying that already :D